lib/cell.rb

```diff
@@ -26,10 +26,10 @@ def fired_upon?
   end
 
   def fire_upon
-    # if @ship.length != 0
-    #
-    # else
-    # end
+    if @ship.length != 0
+      puts "this is confusing"
```

Comment:
This code won't work out, we should change it.

Reply:
Cool let me change that right now